### PR TITLE
First version of EiffelActivityQueuedEvent

### PR DESCRIPTION
**Pull request by danielyinanc**
This is the simple version of an EiffelActivityQueuedEvent. Forms the basis of the schema, makes it easier to explain and understand fields with example.

### CODE REVIEW

File: events/EiffelActivityQueuedEvent/simple.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+      "uri": "http://myhost:8080/additional/information/here"
+    }
+  },
+  "data": {
+    "name": "e.g. Component X Build",
+    "category": "e.g. Component Build",
+    "trigger": {
+      "type": "MESSAGE"
+    },
+    "description": "Triggered by something ",
```
Comment (p-backman-ericsson) [inline events/EiffelActivityQueuedEvent/simple.json]: I think we should try to make examples that could be real. Suggest "Triggered by something " is replaced by "Triggered by EiffelArtifactPublishedEvent"

**Comment by danielyinanc** [comment]
All comments addressed. If no issues, let's merge it.

File: events/EiffelActivityQueuedEvent/simple.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    "version": "1.0",
+    "time": 1234567890,
+    "tags": [
+      "unit-test"
+    ],
+    "source": {
+      "host": "hostname",
+      "pid": 6132,
+      "name": "qwerty",
+      "serializer": {
```
Comment (p-backman-ericsson) [inline events/EiffelActivityQueuedEvent/simple.json]: The serialilzer miss some mandatory fields. If we keep the serializer we should make a GAV that has values that looks real.

File: events/EiffelActivityQueuedEvent/simple.json [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    "category": "Component Build",
+    "trigger": {
+      "type": "MESSAGE",
+      "description": "Triggered by EiffelArtifactPublishedEvent"
+    },
+    "executionType": "AUTOMATED"
+  },
+  "links": {
+    "causes": [
+      "uuid1"
```
Comment (p-backman-ericsson) [inline events/EiffelActivityQueuedEvent/simple.json]: use an uuid that follows the uuid spec.

File: events/EiffelActivityQueuedEvent/simple.json [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+    "type": "EiffelActivityQueuedEvent",
+    "version": "1.0",
+    "time": 1234567890,
+    "tags": [
+      "unit-test"
+    ],
+    "source": {
+      "host": "hostname",
+      "pid": 6132,
+      "name": "qwerty",
```
Comment (p-backman-ericsson) [inline events/EiffelActivityQueuedEvent/simple.json]: Can we invent a better name then "qwerty"?

File: events/EiffelActivityQueuedEvent/simple.json [hunk 5]
```diff
@@ -0,0 +1,26 @@
+{
+  "meta": {
+    "domainId": "domainId-test",
+    "id": "acqueued-ec41-488e-93f2-fc2702b32f21",
```
Comment (p-backman-ericsson) [inline events/EiffelActivityQueuedEvent/simple.json]: To be really picky this is not a valid UUID since the "q" is outside the valid range.

Comment (d-stahl-ericsson) [inline events/EiffelActivityQueuedEvent/simple.json]: With fixed UUID and plain cause link (no type string) this looks good to me.

File: events/EiffelActivityTriggeredEvent/simple.json [hunk 6]
```diff
@@ -0,0 +1,23 @@
+{
+  "meta": {
+    "domainId": "domainId-test",
+    "id": "e1e93f13-7c3c-4f17-9753-ebf0c86ff1c2",
+    "type": "EiffelActivityQueuedEvent",
```
Comment (d-stahl-ericsson) [inline events/EiffelActivityTriggeredEvent/simple.json]: Wrong type :)

File: events/EiffelActivityTriggeredEvent/simple.json [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+    "version": "1.0",
+    "time": 1234567890
+  },
+  "data": {
+    "name": "Component X Build",
+    "category": "Component Build",
+    "trigger": {
+      "type": "MESSAGE",
```
Comment (d-stahl-ericsson) [inline events/EiffelActivityTriggeredEvent/simple.json]: Valid values are MANUAL, EIFFEL_EVENT, SOURCE_CHANGE, TIMER, OTHER.

**Comment by d-stahl-ericsson** [comment]
Very minor: triggerType MANUAL doesn't rhyme well with the description. Other than that, :+1:

**Comment by danielyinanc** [comment]
I do not seem to have the right to merge this, I am not sure why... Can you merge?

**Pull request merged**